### PR TITLE
Move http-static from beefy-scenarios

### PR DESCRIPTION
**Pull request by fedinskiy**
As per https://github.com/quarkus-qe/quarkus-test-suite/issues/91

### CODE REVIEW

**Comment by Sgitario** [comment]
@fedinskiy is this PR ready to be reviewed? If not, pls change it to draft until it is. If so, pls assign anybody as a reviewer.

File: http/http-static/pom.xml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>io.quarkus.ts.qe</groupId>
+        <artifactId>parent</artifactId>
+        <version>1.0.0-SNAPSHOT</version>
+        <relativePath>../..</relativePath>
+    </parent>
+    <artifactId>http-static</artifactId>
+    <packaging>jar</packaging>
+    <name>Quarkus QE TS: static http content</name>
```
Comment (Sgitario) [inline http/http-static/pom.xml:12]: Same comment as in https://github.com/quarkus-qe/quarkus-test-suite/pull/100#discussion_r675316157

File: http/http-static/src/test/java/io/quarkus/ts/http/LargeStaticResourceIT.java [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+import java.net.URL;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+import io.quarkus.test.common.http.TestHTTPResource;
+import io.quarkus.test.junit.QuarkusTest;
+import io.restassured.response.Response;
+
+@QuarkusTest
```
Comment (Sgitario) [inline http/http-static/src/test/java/io/quarkus/ts/http/LargeStaticResourceIT.java:17]: This test is not using the test framework:
1.- Change it to `@QuarkusScenario`
2.- This annotation `@TestHTTPResource("big-file")` is not working in the test framework (you can file an issue to support it). Instead, use directly given().post("/big-file"),...

**Review by Sgitario — CHANGES_REQUESTED** [review]
Please, link the related issue to this PR and squash all the commits in one.

**Comment by fedinskiy** [comment]
@Sgitario issue linked, commits squashed, all requested changes was implemented

**Review by Sgitario — APPROVED** [review]
lgtm

**Comment by Sgitario** [comment]
The failure is unrelated to this PR. Merging.

**Pull request merged**